src/main/java/org/testeditor/fixture/host/locators/LocatorByStart.java

```diff
@@ -88,32 +134,35 @@ public int getStartRow() {
      *         startRow, startColumn.
      */
     private LocatorByStart createLocatorByStart(String elementLocator) {
```

Comment:
The name of this method is misleading. Also the implementation is kind of hacky... here is an alternative:
```
    private Pattern locatorPattern = Pattern.compile("(?<startRow>\\d+);(?<startColumn>\\d+)");
    
    private void initializeStartRowAndColumn(String elementLocator) {
    	Matcher matcher = locatorPattern.matcher(elementLocator);
    	if (matcher.matches()) {
    		this.startRow = Integer.parseInt(matcher.group("startRow"));
    		this.startColumn = Integer.parseInt(matcher.group("startColumn"));
    		this.startRowWithOffset = this.startRow + this.offsetRow;
    		this.startColumnWithOffset = this.startColumn + this.offsetColumn;
    		checkBoundaries();
    	} else {
    		throw new IllegalArgumentException("The provided locator did not match the expected pattern \"x;y\" where x and y are both integer values. Got: " + elementLocator);
    	}
    }
```